client/src/document/ingredients/browser-compatibility-table/utils.ts

```diff
@@ -112,3 +115,45 @@ export function isFullySupportedWithoutLimitation(
 export function isNotSupportedAtAll(support: bcd.SimpleSupportStatement) {
   return !support.version_added && !hasLimitation(support);
 }
+
+function isFullySupportedWithMinorLimitation(
+  support: bcd.SimpleSupportStatement
+) {
+  return support.version_added && !hasMajorLimitation(support);
+}
+
+export function getCurrent<T>(a: T | T[]): T;
+// Prioritizes support items
+export function getCurrent(support: bcd.SupportStatement | undefined) {
```

Comment:
getCurrent() only works with support statements, and we can merge the type declaration with the function declaration:

```suggestion
// Prioritizes support items
export function getCurrent(support: bcd.SupportStatementExtended | undefined): bcd.SimpleSupportStatementExtended | undefined {
```

Also, since `getFirst()` is type-agnostic, it would probably make sense to rename this to `getCurrentSupport()`.

Reply:
Namespace '".../yari/node_modules/@mdn/browser-compat-data/types"' has no exported member 'SupportStatementExtended'.

SupportStatementExtended is defined by: https://github.com/mdn/yari/blob/38f3f0b99b166068acb8d51a20f92cec68b386bc/client/src/document/ingredients/browser-compatibility-table/feature-row.tsx#L36-L38

This was doable, but it needed a more extensive change.